### ISSUE TITLE
Skip certain folders when searching for interpreters and environments

### ISSUE THREAD

**Issue by petetronic** [posit-dev/positron#5664]
Feedback has shown that Positron on latest Mac OS can prompt users for access to their Music folder unintentionally. It may be that the search for interpreters and environments is causing this. We should confirm and skip certain folders, such as ~/Music (and others to be determined), as we should not expect to look in these locations.

**Comment by ntluong95** [comment]
I dont like the "searching interpreter" function. It takes a lot of time when starting. Why not just do it for the first time installing Positron, and let user pre-define the default interpreter path. If they want to change, they do it manually. I suppose R user usually not creating a virtual environment, so just use the global version. For python user, they can set the default interpreter and change it at anytime

**Comment by juliasilge** [comment]
We are planning on work in early 2025 to provide users more control on how they specify interpreters. We've got a very initial description in https://github.com/posit-dev/positron/issues/3574, if you'd like to add more details @ntluong95.

**Comment by juliasilge** [comment]
Seems mostly likely this is the Python discovery but we don't know that as of today.

**Comment by juliasilge** [comment]
We got a report on this from @mine-cetinkaya-rundel after she updated her OS.

**Comment by testlabauto** [comment]
Came up again in a discussion: https://github.com/posit-dev/positron/discussions/14339

**Comment by jmcphers** [comment]
Of course, if we skip these folders we're going to find out that hundreds of people install Python into `~/Music` and want to know why we aren't finding it...
